--- src/Makefile.in.old	2019-05-23 13:33:29.715557552 +0300
+++ src/Makefile.in	2019-05-23 13:33:56.788713369 +0300
@@ -166,8 +166,8 @@
 # Installation.  DESTDIR can be used to modify the install location.
 ####################################################################################################################################
 install: pgbackrest
-	install -d $(DESTDIR)
-	install -m 755 pgbackrest $(DESTDIR)
+	install -d $(DESTDIR)/usr/gbd/pgbackrest/bin
+	install -m 755 pgbackrest $(DESTDIR)/usr/gbd/pgbackrest/bin
 
 ####################################################################################################################################
 # Uninstallation.  DESTDIR should be set to the same value as when installed.
