--- src/Makefile.old	2019-04-14 02:28:17.649284151 +0100
+++ src/Makefile	2019-04-14 02:29:02.620557231 +0100
@@ -182,8 +182,8 @@
 # Installation.  DESTDIR can be used to modify the install location.
 ####################################################################################################################################
 install: pgbackrest
-	install -d $(DESTDIR)
-	install -m 755 pgbackrest $(DESTDIR)
+	install -d $(DESTDIR)/usr/gbd/pgbackrest/bin
+	install -m 755 pgbackrest $(DESTDIR)/usr/gbd/pgbackrest/bin
 
 ####################################################################################################################################
 # Uninstallation.  DESTDIR should be set to the same value as when installed.
