--- src/Makefile.old	2019-01-13 23:41:32.143678524 +0300
+++ src/Makefile	2019-01-13 23:41:49.886780081 +0300
@@ -156,8 +156,8 @@
 # Installation.  DESTDIR can be used to modify the install location.
 ####################################################################################################################################
 install: pgbackrest
-	install -d $(DESTDIR)/usr/bin
-	install -m 755 pgbackrest $(DESTDIR)/usr/bin
+	install -d $(DESTDIR)/usr/gbd/pgbackrest/bin
+	install -m 755 pgbackrest $(DESTDIR)/usr/gbd/pgbackrest/bin
 
 ####################################################################################################################################
 # Compile rules
